apps/wakunode2/app.nim

```diff
@@ -570,7 +570,7 @@ proc startRestServer(app: App, address: ValidIpAddress, port: Port, conf: WakuNo
   ## Relay REST API
   if conf.relay:
     let relayCache = TopicCache.init(capacity=conf.restRelayCacheCapacity)
-    installRelayApiHandlers(server.router, app.node, relayCache)
+    installRelayApiHandlers(server.router, app.node, app.wakuDiscv5, relayCache)
```

Comment:
It seems odd to pass here `wakuDiscv5`. Perhaps `wakuDiscv5` should be part of the `node`.

Not a blocker though.

Reply:
It is weird but it's the simplest thing I could do.

A refactor of how handlers send info. internally would be good.